src/components/Icons/ArrowCircleLeft/ArrowCircleLeft.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { SVG } from '../Icons.styles';
+
+const ArrowCircleLeft = () => {
+  return (
+    <>
```

Comment:
You don't need this fragment, as there's only element. Can you check this across all files please.

Reply:
All sorted